cmd/envtool/envtool.go

```diff
@@ -244,11 +244,12 @@ func main() {
 	}
 
 	opts := &logging.NewHandlerOpts{
-		Base:          "console",
-		Level:         level,
-		RemoveTime:    true,
-		RemoveSource:  true,
-		CheckMessages: false, // TODO https://github.com/FerretDB/FerretDB/issues/4511
+		Base:         "console",
+		Level:        level,
+		RemoveTime:   true,
+		RemoveSource: true,
+		// the logger handles messages from the `go test` command output
+		CheckMessages: false,
```

Comment:
I'm not quite sure why envtool handles `go test` output. :thinking: I think it should be possible to enable this one.

Reply:
envtool runs `go test` as well, as some other shell commands. It logs their stdout.